nss-day-cohort-44/holiday-road-isle-royale
# css layout changes

# Description
formatted the css for the main container with flexbox
## Type of change

# Testing Instructions
1. `git fetch --all`
1. `git checkout sl-start2`
1. `serve`
1.  `code .`
1. check layout in localhost
